examples/production/values.yaml

```diff
@@ -47,10 +67,39 @@ spire-server:
       seccompProfile:
         type: RuntimeDefault
 
+    resources:
+      requests:
+        cpu: 100m
+        memory: 64Mi
+      limits:
+        cpu: 200m
+        memory: 256Mi
+
+    ignoreNamespaces:
+      - kube-system
+      - kube-public
+      - local-path-storage
+
 spiffe-csi-driver:
   enabled: true
   namespaceOverride: spire-system
-  priorityClassName: system-node-critical
+
+  resources:
+    requests:
+      cpu: 100m
+      memory: 32Mi
+    limits:
+      cpu: 200m
+      memory: 128Mi
+
+  nodeDriverRegistrar:
```

Comment:
This chunk should be removed. we dont have it.

Reply:
We do have that see the daemonset. There is a second container.

Reply:
Doh. I misread that and was thinking about the other k8s sync thing. Sorry.